test/Microsoft.AspNetCore.Server.Kestrel.FunctionalTests/ResponseTests.cs

```diff
@@ -2063,39 +2063,34 @@ public Task ResponseStatusCodeSetBeforeHttpContextDisposedRequestMalformedReadIg
 
         [Theory]
         [MemberData(nameof(ConnectionAdapterData))]
-        public async Task FailedWritesResultInAbortedRequest(ListenOptions listenOptions)
+        public async Task ThrowsOnWriteAfterRequestIsAborted(ListenOptions listenOptions)
```

Comment:
Nit: Maybe ThrowsOnWriteWithRequestAbortedTokenAfterRequestIsAborted since normally we don't throw on write after a request is aborted.